utils/utils.go

```diff
@@ -14,11 +14,17 @@ import (
 )
 
 func RepoDir() string {
-	cacheDir, err := os.UserCacheDir()
-	if err != nil {
-		log.Fatalln("gibo can't determine your user cache directory. Please file an issue at https://github.com/simonwhitaker/gibo/issues")
+	xdgCacheDir := os.Getenv("XDG_CACHE_HOME")
```

Comment:
What's your intention here? `os.UserCacheDir()` will respect `XDG_CACHE_HOME` on Unix systems. I suppose people on macOS or Windows might set the `XDG_CACHE_HOME` environment variable, but I don't think it makes sense to try to honour non-standard configs; the logic here would potentially get very complicated, for little benefit.

Reply:
I understood. New commit:[13959a970eda9e4c61afe6105f176c01f9d14ff5](https://github.com/simonwhitaker/gibo/pull/63/commits/13959a970eda9e4c61afe6105f176c01f9d14ff5) remove these codes relied XDG_CACHE_HOME.
please check them.